src/src/hw7/hillel/Animals.java

```diff
@@ -0,0 +1,41 @@
+package hw7.hillel;
+
+public class Animals {
+    private static int AnimalsCount;
+    private String name;
+    private int runnig;
+    private int swiming;
+
+
+
+    public void swim(int swiming) {
+        System.out.println("Animals swim:" + swiming);
+    }
+
+    public void run(int runnig) {
+        System.out.println("Animals run:" + runnig);
+    }
+
+    public Animals(String name) {
```

Comment:
Class members should have the following order:
1. Constants
2. static variables
3. instance variables
4. constructors
5. getters + setters
6. public methods
8. private methods
Please change your class member's order accordingly.